### PR TITLE
Curative - only add date if there are appointments

### PR DESCRIPTION
**Pull request by zpeyton**
This: 
```availability: {
        '03/04/2021': { numberAvailableAppointments: 0, hasAvailability: false },
        '03/05/2021': { numberAvailableAppointments: 1, hasAvailability: true },
        '03/06/2021': { numberAvailableAppointments: 0, hasAvailability: false },
        '03/07/2021': { numberAvailableAppointments: 0, hasAvailability: false },
        '03/08/2021': { numberAvailableAppointments: 0, hasAvailability: false },
        '03/09/2021': { numberAvailableAppointments: 0, hasAvailability: false },
        '03/10/2021': { numberAvailableAppointments: 0, hasAvailability: false },
        '03/11/2021': { numberAvailableAppointments: 0, hasAvailability: false },
        '03/12/2021': { numberAvailableAppointments: 0, hasAvailability: false },
        '03/13/2021': { numberAvailableAppointments: 0, hasAvailability: false },
        '03/14/2021': { numberAvailableAppointments: 0, hasAvailability: false }
      }
```

should probably be 

```
availability: {
        '03/05/2021': { numberAvailableAppointments: 1, hasAvailability: true  }
      }
```

### CODE REVIEW

**Comment by tkjones117** [comment]
Does this affect functionality or just clean up the data?

**Review by harcod — APPROVED** [review]
LGTM. I successfully ran this PR locally.

File: site-scrapers/Curative/index.js [hunk 1]
```diff
@@ -61,10 +61,13 @@ module.exports = async function GetAvailableAppointments(browser) {
...
 
-            mappedData.availability[date] = {
-                numberAvailableAppointments: newNumberAvailable,
-                hasAvailability: !!newNumberAvailable,
-            };
+            // Only add date keys if there are appointments
+            if (newNumberAvailable) {
+                mappedData.availability[date] = {
+                    numberAvailableAppointments: newNumberAvailable,
+                    hasAvailability: !!newNumberAvailable,
```
Comment (tkjones117) [inline site-scrapers/Curative/index.js:68]: you could simplify to `hasAvailability: true`, no?

**Comment by zpeyton** [comment]
> Does this affect functionality or just clean up the data?

@tkjones117 Just clean up

**Review by tkjones117 — APPROVED** [review]

**Review by harcod — APPROVED** [review]

**Pull request merged**